src/components/Forms/OrderForm.tsx

```diff
@@ -64,11 +64,12 @@ const OrderFormSchema = Yup.object().shape<OrderSchemaType>({
 
 interface Props {
   defaultOrderData?: OrderData;
+  setShippingMethod: React.Dispatch<React.SetStateAction<number | undefined>> 
 }
 
 function OrderForm(props: Props) {
   let navigate = useNavigate();
-  const [setOrderDetails] = useLocalStorage("orderDetails", "");
+  const [,setOrderDetails] = useLocalStorage<OrderData>("orderDetails", "");
```

Comment:
räckte med ',' där alltså?

Reply:
Precis! Det krävs 2 värden men man kan även bara lägga in ett komma i början, då slipper man alla oanvända värden i början :)   (kan säkert användas på fler ställen vid uppstäd)